src/content/components/FeatureList/FeatureList.js

```diff
@@ -97,7 +98,13 @@ export class FeatureList extends React.PureComponent {
 
   render() {
     const release = Number(
-      this.context.iterations.getIteration().number.split(".")[0]
+      this.context.iterations.getIteration()?.number.split(".")[0] ??
+        this.context.iterations
+          .getAdjacentIteration(
```

Comment:
not sure what's going on here, this seems to be getting the second-latest iteration, not the latest iteration